packages/kotti-ui/source/kotti-comment/components/CommentReply.vue

```diff
@@ -1,131 +1,103 @@
-						<KtButton icon="check" @click="handleConfirm" />
-					</KtButtonGroup>
-				</div>
-				<div
-					v-if="!isInlineEdit & (actionOptions.length > 0)"
-					class="comment-reply__action action__more"
-				>
-					<i class="yoco">dots</i>
-					<div class="action__options">
-						<a
-							v-for="(option, index) in actionOptions"
-							:key="index"
-							@click="option.onClick"
-						>
-							<li v-text="option.label" />
-						</a>
-					</div>
-				</div>
+		<div class="comment-reply__content">
+			<div class="comment-reply__content__info">
+				<div class="info__name" v-text="userName" />
+				<div class="info__time" v-text="createdTime" />
 			</div>
+
+			<CommentInlineEdit
+				:id="id"
+				:dangerouslyOverrideParser="dangerouslyOverrideParser"
+				:isEditing="isEditing"
+				:message="message"
+				:postEscapeParser="postEscapeParser"
+				@edit="($event) => $emit('edit', $event)"
+				@update:isEditing="($event) => (isEditing = $event)"
+			/>
+
+			<CommentActions
+				:options="actionOptions"
+				:userData="{ userId, userName }"
+				@replyClick="($event) => $emit('click', $event)"
+			/>
 		</div>
 	</div>
 </template>
 
-<script>
+<script lang="ts">
+import { computed, defineComponent, ref } from '@vue/composition-api'
 import escape from 'lodash/escape'
 
-import { KtAvatar } from '../../kotti-avatar'
-import { KtButton } from '../../kotti-button'
-import { KtButtonGroup } from '../../kotti-button-group'
+import { Kotti } from '../../types'
 
-export default {
+import CommentActions from './CommentActions.vue'
+import CommentInlineEdit from './CommentInlineEdit.vue'
+
+export default defineComponent<Kotti.Comment.Reply.PropsInternal>({
 	name: 'CommentReply',
 	components: {
-		KtAvatar,
-		KtButton,
-		KtButtonGroup,
+		CommentActions,
+		CommentInlineEdit,
 	},
 	props: {
-		createdTime: String,
+		createdTime: { default: () => null, type: String },
 		dangerouslyOverrideParser: { default: escape, type: Function },
 		isDeletable: { default: false, type: Boolean },
 		isEditable: { default: false, type: Boolean },
-		id: [Number, String],
-		message: String,
+		id: { default: () => null, type: [Number, String] },
+		message: { type: String, required: true },
 		parser: { default: escape, type: Function },
 		postEscapeParser: { default: (_) => _, type: Function },
-		userAvatar: String,
-		userId: [Number, String],
-		userName: String,
+		userAvatar: { default: () => null, type: String },
+		userId: { default: () => null, type: Number },
+		userName: { default: () => null, type: String },
 	},
-	data() {
+	setup(props, { emit }) {
+		const isEditing = ref<boolean>(false)
+
 		return {
-			inlineMessageValue: '',
-			isInlineEdit: false,
+			actionOptions: computed(() => {
+				const options = []
+				if (props.isEditable)
+					options.push({
+						label: 'Edit',
+						onClick: () => {
+							isEditing.value = true
+						},
+					})
+				if (props.isDeletable)
+					options.push({
+						label: 'Delete',
```

Comment:
Shouldn't this use translation strings?
`label: translations.value.deleteButton,`